_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
+            }
+            if (data[i]["in_foundry_order"]) {
+                foundry_order = `
+                        <a class="col-sm-1 btn btn-default btn-sm" href="/principles/fp-000-summary.html" aria-label="View the OBO Foundry criteria for the Foundry status of ${title}" title="View the OBO Foundry criteria for the Foundry status of ${title}">
+                               <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                        </a>             
+                `;
+            }
+            if (activity_status === "inactive") {
+                is_inactive = "inactive_row";
+            }
+            if (data[i]["is_obsolete"]) {
+                is_obsolete = "(obsolete)"
+            }
+            // table row template
+            let template = `
+                <tr class="row ${is_inactive}">
+                    <td class="col-sm-1">
+                        <a class="" href="ontology/${id}.html">
+                           ${id} 
+                        </a>
+                        <span style="background-color: #ff8d82">${is_obsolete}</span>    
+                    </td>
+                    <td class="col-sm-1">
+                        ${title}
+                    </td>
+                    <td class="col-sm-3">
+                        ${description}
+                        <small>
+                            <a href="ontology/${id}.html">Detail</a>
+                        </small>
+                    </td>
+                    <td class="col-sm-5">
+                        <div class="ic-display">
+                            <a class="btn btn-default btn-sm" href="ontology/${id}.html" aria-label="More details for ${title}" title="More details for ${title}">
+                                  <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="${homepage}" aria-label="Go to the homepage for ${title}" title="Go to the homepage for ${title}">
+                                 <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="http://purl.obolibrary.org/obo/${id}.owl" aria-label="Download ${title} in OWL format" title="Download ${title} in OWL format">
+                                <span class="glyphicon glyphicon-download" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="http://www.ontobee.org/browser/index.php?o=${id}" aria-label="Browse ${title} on OntoBee" title="Browse ${title} on OntoBee">
+                                <span class="glyphicon glyphicon-eye-open" aria-hidden="true"></span>
+                            </a>
+                            ${tracker}
+                            ${contact}
+                            ${publication}
```

Comment:
this seems to show an icon even if there's no publication

Reply:
Cant confirm this, looks correct on the site. If you observe this in production, add to https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1994